test/services/users.test.js

```diff
@@ -37,28 +26,20 @@ describe('User service tests', function () {
       .insert(user)
       .then(function (newUser) {
         let expectedUserProps = ['firstname', 'lastname', 'email', 'isActive', 'username', 'id', 'createdAt', 'updatedAt'];
-        /* including what's predictable. id, createdAt and updatedAt are initialized at
-        time of creation, so they're not predictable */
-        let expectedPartialUserObj = {
-          firstname: user.firstname,
-          lastname: user.lastname,
-          username: user.username,
-          email: user.email,
-          isActive: true
-        };
         Object.keys(newUser).should.eql(expectedUserProps);
-        should.deepEqual(_.omit(newUser, ['id', 'createdAt', 'updatedAt']), expectedPartialUserObj);
-        db.hgetallAsync(redisConfig.namespace.concat('-', 'user').concat(':', newUser.id))
+        newUser.firstname.should.eql(user.firstname);
+        newUser.lastname.should.eql(user.lastname);
+        newUser.username.should.eql(user.username);
```

Comment:
dtto - [Assertion properties](http://shouldjs.github.io/#assertion-properties) are probably a better tool for this.

Reply:
👍